doc/posts/2023/2023_06_19_vara_week4.rst

```diff
@@ -0,0 +1,18 @@
+Diffusion research continues: Week 4
+====================================
+
+.. post:: June 19, 2023
+     :author: Vara Lakshmi Bayanagari
+     :tags: google
+     :category: gsoc
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
Space required after this. I'm recommending this because Serge recommended it on my blog post.